inference-engine/src/vpu/myriad_plugin/myriad_metrics.cpp

```diff
@@ -110,11 +110,10 @@ const std::unordered_set<std::string>& MyriadMetrics::OptimizationCapabilities()
 }
 
 std::string MyriadMetrics::DeviceArchitecture(const std::map<std::string, InferenceEngine::Parameter> & options) const {
-    // TODO: Task 49309. Return same architecture for devices which can share same cache
-    // E.g. when device "MYRIAD.ma2480-1" is loaded, options.at("DEVICE_ID") will be "ma2480-1"
-    // For DEVICE_ID="ma2480-0" and DEVICE_ID="ma2480-1" this method shall return same string, like "ma2480"
-    // In this case inference engine will be able to reuse cached model and total reduce load network time
-    return "MYRIAD";
+    std::string nameDelimiter("-ma");
```

Comment:
Should we use more complicated logic, like it is done in MyriadMetrics::FullName? @Maxim-Doronin maybe you can comment?